src/utils/patient-programs-helper.ts

```diff
@@ -0,0 +1,4 @@
+export const getPatientPrograms = (patient: any, programUuid: Record<string, any>) => {
+  const patientPrograms = patient.patientPrograms;
+  return patientPrograms.find((program) => program.program.uuid == programUuid);
```

Comment:
Do we need a new file for this?

Reply:
I can move this to the `EncounterFormManager` class